main/EventSystem.cpp

```diff
@@ -2488,6 +2488,21 @@ bool CEventSystem::parseBlocklyActions(const _tEventItem &item)
 			actionsDone = true;
 			continue;
 		}
+		else if (deviceName.find("CustomCommand:") == 0)
```

Comment:
The colon isn't used when parsing Blockly and Python. You also forgot to set afterTimerSeconds in the AddTaskItem line.

Reply:
Hm, I note we are inconsistent about whether it's `SetSetPoint:` or `SetSetpoint` though. Lua uses the former while Blockly and Python use the latter.

Reply:
`$reasons_to_unify_this_stuff++;` :)

Reply:
Should be Setpoint (one word, see wikipedia)

Reply:
We'll probably have to make it accept both now, for compatibility. But that's all for a different PR anyway.

Reply:
i agree, but should be changed soon else we keep using both ;)